salt/modules/mssql.py

```diff
@@ -195,19 +229,28 @@ def role_create(role, owner=None, **kwargs):
 
         salt minion mssql.role_create role=product01 owner=sysdba
     '''
+    if not grants:
```

Comment:
As soon as this requested change is made, I think we should be able to merge this in.